<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogueClass</class>
 <widget class="QWidget" name="DialogueClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialogue</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>541</width>
     <height>421</height>
    </rect>
   </property>
   <widget class="QLabel" name="Reponse4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 150);
color : white;
font-weight : bold;
font-size : 14px;
padding-left: 5px;
padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>Bon Ok.voilà un arbre qui parle...Après tout, moi aussi je parle, alors pourquoi pas lui?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRep3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 0);
color : white;
font-weight : bold;
font-size : 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRep4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 0);
color : white;
font-weight : bold;
font-size : 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Reponse3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 150);
color : white;
font-weight : bold;
font-size : 14px;
padding-left: 5px;
padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>Un arbre qui parle. Non mais c'est n'importe quoi !</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Reponse1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 150);
color : white;
font-weight : bold;
font-size : 14px;
padding-left: 5px;
padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>Bon Ok.voilà un arbre qui parle...Après tout, moi aussi je parle, alors pourquoi pas lui?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRep2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 0);
color : white;
font-weight : bold;
font-size : 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Question">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 150);
color : white;
font-weight : bold;
font-size : 14px;</string>
    </property>
    <property name="text">
     <string>Qui ose me déranger durant ma sieste? Ah... c'est vous ! Cela faisait longtemps que je vous attendais. Permettez-moi de me présenter. Je suis le Chêne de Lafayette. On dit que j'ai été planté ici au XVII siècle par le grand père du Général Lafayette, mais je vous avoue que je ne me souviens pas très bien de mes premières années.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Reponse2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 150);
color : white;
font-weight : bold;
font-size : 14px;
padding-left: 5px;
padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>Mais ?! Tu parles !</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRep1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 0);
color : white;
font-weight : bold;
font-size : 14px;</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>Reponse4</zorder>
   <zorder>Reponse3</zorder>
   <zorder>Reponse1</zorder>
   <zorder>Question</zorder>
   <zorder>Reponse2</zorder>
   <zorder>btnRep1</zorder>
   <zorder>btnRep2</zorder>
   <zorder>btnRep3</zorder>
   <zorder>btnRep4</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Dialogue.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnRep1</sender>
   <signal>pressed()</signal>
   <receiver>DialogueClass</receiver>
   <slot>answer1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRep2</sender>
   <signal>pressed()</signal>
   <receiver>DialogueClass</receiver>
   <slot>answer2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRep3</sender>
   <signal>pressed()</signal>
   <receiver>DialogueClass</receiver>
   <slot>answer3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRep4</sender>
   <signal>pressed()</signal>
   <receiver>DialogueClass</receiver>
   <slot>answer4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>answer1()</slot>
  <slot>answer2()</slot>
  <slot>answer3()</slot>
  <slot>answer4()</slot>
 </slots>
</ui>
